### DISCUSSION TITLE
Defining wind speed to be parallel to a slope

### DISCUSSION THREAD

**Discussion by ahmedahmed2056** [firemodels/fds#13233] [General]
![image](https://github.com/user-attachments/assets/a62e30c8-9814-459b-aa52-c2853a43ef36)
I want to define the transient wind speed to be parallel to this slope .. I did wind direction ramp with time but it seems that the wind direction is controlling the direction in the xy plan only 

I tried to read in the FDS guide and i quote the following 
![image](https://github.com/user-attachments/assets/88e63a50-08b5-4986-a17f-0015bbad5add)

So how to change the direction to be sloped?

**Comment by mcgratta** [comment]
Model the ramp, house, and all surroundings such that you can specify the wind at the external boundary. Do not try to model the ramp and house only because the wind field will not develop naturally in such a case.

**Reply by ahmedahmed2056** [reply]
Can you please explain your answer more? I couldn't get your point?

**Reply by ahmedahmed2056** [reply]
Because the profile of temperature in this paper is changing around the slope axis. I guess that means that it changes in the z direction.

**Reply by mcgratta** [reply]
Wind is imposed at the mesh boundary. You cannot direct it within the mesh; only at the boundary. I am suggesting that you widen the computational domain to include the hill, house and all other surrounding terrain. In that way, the wind field imposed at the boundary will develop naturally within the domain.

**Reply by ahmedahmed2056** [reply]
so widening the computational domain to wide enough will make the wind adapt and develop naturally to go with the slope and interact with the surrounding ? interesting
